scripts/check-data-freshness.ts

```diff
@@ -53,7 +23,19 @@ test('All URLs linking to current programs have an OK response code', async tap
         console.error('${url_to_check} is not a valid URL for program: ', key);
         process.exit(1);
       } else {
-        const status = await checkUrlDataAvailability(url_to_check);
+        const response = await returnAvailableUrlData(
```

Comment:
Optional rename (and you can do this as a follow-up when this is all over if it's easier) to something like `retrieveWebContent`.

Reply:
Very cool suggestion, thanks! Done